alfresco-transformer-base/src/main/java/org/alfresco/transformer/config/GlobalProperties.java

```diff
@@ -0,0 +1,81 @@
+package org.alfresco.transformer.config;
```

Comment:
The T-Base should include anything specific to external transformers  since it is also used to build custom T-Engines (that are unrelated to I/M, L/O, P/R).